### ISSUE TITLE
Color background in SMS and SGG emulation - version 2.4 and 2.6.2

### ISSUE THREAD

**Issue by andrzejlisek** [TASVideos/BizHawk#2892]
If I run any Sega Master System or Sega GameGear game, the background outside the picture viewport has color other than black and the color changes during the game, also when the display overscan is set to false in settings. If I run the game for other console than SMS or SGG, like NES/SNES/GB/GBC/GBA/MegaDrive, the background outside game viewport is black as it should be.

How to set the background to black in Sega Master System and Sega Game Gear?

![Zrzut ekranu z 2021-08-02 16-34-37](https://user-images.githubusercontent.com/33282976/127879194-fa0547f8-51b1-43d2-a7aa-3a0a860d5793.png)
![Zrzut ekranu z 2021-08-02 16-35-32](https://user-images.githubusercontent.com/33282976/127879203-09cd4572-39d7-41e1-beb5-cc54142aed49.png)

The problem is in version 2.4 in Windows 8 and in version 2.6.2 on Ubuntu Linux.

**Comment by vadosnaprimer** [comment]
How does it look on actual console?

**Comment by andrzejlisek** [comment]
> How does it look on actual console?

I checked and it has really colored border. But I supposed, that the colored border is overscan and the displays only when "Sisplay Overscan" is set to true in settings.

**Comment by alyosha-tas** [comment]
It looks like maybe you manually scaled your screen? That can give extra boarder region even if you have overscan off. Try going to View -> Window Size and picking a pre-determined scale factor.

But I guess I can make boarder area black if overscan is turned off as well.

**Comment by andrzejlisek** [comment]
> It looks like maybe you manually scaled your screen? That can give extra boarder region even if you have overscan off. Try going to View -> Window Size and picking a pre-determined scale factor.

If I use the View -> Window Size, there is not border. In Config -> Display I checked the "Expand pixens by integer only" and "Use 1:1 pixel size". Really, I manually resized window, but I will get the same effect (color border) if I maximize the window or switch into fullscreen.

> But I guess I can make boarder area black if overscan is turned off as well.

I also supposed, that the emulator should not draw anything or change colors of the border, outside the emulated screen. If I turn the "Display Overscan" to true in SMS -> Settings, I will have extra border inside the viewport (the viewport will be enlarged).

> EDIT; actually, the undrawn area should just be black the overscan area needs to updated as demonstrated in #2816

Yes, if the  "Display Overscan" is true, the screen viewport will be slightly enlarged and should be colored as in actual console, but the border outside the viewport should be always black.
